foreman/foreman.spec

```diff
@@ -190,6 +190,8 @@ BuildRequires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 BuildRequires: %{?scl_prefix_ror}rubygem(execjs) < 2.5.0
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+BuildRequires: %{?scl_prefix_ror}rubygem(autoprefixer-rails) >= 5.2
```

Comment:
a) scl_prefix as it's in our collection
b) layout should match assets.rb, move below jquery-ui-rails